leapp.spec

```diff
@@ -0,0 +1,121 @@
+Name:       leapp
+Version:    0.0.1
+Release:    3
+Summary:    leapp tool rpm
+
+Group:      Unspecified
+License:    GPLv2+
```

Comment:
Yeah I know, but I had to put something in - I noticed later that you put there LGPL  

Anyway the COPYING file says it more clearly - It's TBD